wcfsetup/install/files/lib/http/middleware/RedirectMediaToFrontend.class.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
Why are you using an extra middleware for a basic redirect that could just be implemented in the existing `wcf\acp\page\MediaPage`?